packages/manager/modules/enterprise-cloud-database/src/service/details/cluster-size/cluster-size.routing.js

```diff
@@ -0,0 +1,32 @@
+export default /* @ngInject */($stateProvider) => {
+  $stateProvider.state('enterprise-cloud-database.service.details.cluster-size', {
+    component: 'enterpriseCloudDatabaseServiceDetailsClusterSizeComponent',
+    resolve: {
+      addReplicas: /* @ngInject */ ($state, clusterId, hosts) => callback => $state.go('enterprise-cloud-database.service.details.cluster-size.add-replicas', { clusterId, hostList: hosts, callback }),
+      deleteReplicas: /* @ngInject */ ($state, clusterId) => () => $state.go('enterprise-cloud-database.service.details.cluster-size.delete', { clusterId }),
+    },
+    translations: {
+      value: ['.'],
+      format: 'json',
+    },
+    url: '/cluster-size',
+  }).state('enterprise-cloud-database.service.details.cluster-size.add-replicas', {
```

Comment:
those states should be declared separately
`enterprise-cloud-database.service.details.cluster-size` is in the right file but `enterprise-cloud-database.service.details.cluster-size.add-replicas` should be in a `.../cluster-size/add-replicas/add-replicas.routing.js` as part of the `addReplicas` module

Reply:
This is fixed now. Thanks